docs/pages/versions/v44.0.0/sdk/filesystem.md

```diff
@@ -193,6 +193,8 @@ So, for example, the URI to a file named `'myFile'` under `'myDirectory'` in the
 
 Expo APIs that create files generally operate within these directories. This includes `Audio` recordings, `Camera` photos, `ImagePicker` results, `SQLite` databases and `takeSnapShotAsync()` results. This allows their use with the `FileSystem` API.
 
+Some `FileSystem` functions are able to read from (but not write to) other locations. Currently `FileSystem.getInfoAsync()` and `FileSystem.copyAsync()` are able to read from URIs returned by [`CameraRoll.getPhotos()`](https://reactnative.dev/docs/cameraroll.html#getphotos) from React Native.
```

Comment:
I did not get why this is being mentioned in SDK 44 docs?

Reply:
> I did not get why this is being mentioned in SDK 44 docs?

If you look at the current SDK 44 docs, you'll see that this paragraph is currently present there exactly as described here. The reviewed snippet you have here is (I presume) from one of my specific commits on this PR, and what I did here is just "bring back" this paragraph because I mistakenly removed it in one of the earlier commits. However, upon further review, I see that I forgot to remove the sentence which mentions `CameraRoll` in SDK 44 docs in that specific paragraph. I'll make the necessary changes